app/helpers/react_components/editor.rb

```diff
@@ -18,7 +18,8 @@ def to_s
           ),
           submission: SerializeSubmission.(solution.submissions.last),
           files: SerializeFiles.(solution.editor_solution_files),
-          language: solution.editor_language
+          language: solution.editor_language,
+          storage_key: solution.uuid
```

Comment:
I recommend prefixing it so you can tell from the prefix what is stored. For example `solution-files-${uuid}`.